cognite_toolkit/_cdf_tk/commands/featureflag.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import annotations
+
+import tempfile
+from enum import Enum
+from functools import lru_cache
+from pathlib import Path
+from typing import Any, ClassVar
+
+import yaml
+from rich import print
+from rich.table import Table
+
+from cognite_toolkit._cdf_tk.commands._base import ToolkitCommand
+from cognite_toolkit._cdf_tk.exceptions import ToolkitRequiredValueError
+
+
+class Flags(Enum):
+    INTERACTIVE_INIT: ClassVar[dict[str, Any]] = {"visible": True, "description": "Enables interactive init"}
+    INTERNAL: ClassVar[dict[str, Any]] = {"visible": False, "description": "Does nothing"}
+
+
+class FeatureFlag:
```

Comment:
General comment on this class, I would not make the methods that are used outside private. For example, `_load_user_settings` used in `FeatureFlagCommand`.